keanu-python/keanu/inference.py

```diff
@@ -5,16 +5,14 @@
 k = KeanuContext().jvm_view()
 
 java_import(k, "io.improbable.keanu.network.BayesianNetwork")
-
+java_import(k, "io.improbable.keanu.algorithms.mcmc.MetropolisHastings")
 
 class BayesNet(JavaObjectWrapper):
     def __init__(self, vertices):
+        self.vertices = vertices
```

Comment:
If we just need it for tests, I think it'll be better if we just call Java's `.getVertices().size()` directly?